src/handlers/primitive-handler-abstract.ts

```diff
@@ -0,0 +1,33 @@
+import FakerStatic = Faker.FakerStatic;
+import { PropertyDto } from '../types/property-dto.interface';
+import { ExactValue } from '../types/exact-value.type';
+
+export abstract class PrimitiveHandlerAbstract {
+  protected static readonly PRIMITIVES = ['String', 'Boolean', 'Number', 'Date'];
```

Comment:
extract this
PRIMITIVES = ['String', 'Boolean', 'Number', 'Date'];

to some outside const

Reply:
Why?

Reply:
I saw 2 copies of this array in 2 files
  protected static readonly PRIMITIVES = ['String', 'Boolean', 'Number', 'Date'];